sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -225,6 +225,18 @@ static const struct dmi_system_id sof_rt5682_quirk_table[] = {
 					SOF_RT5682_SSP_AMP(2) |
 					SOF_RT5682_NUM_HDMIDEV(4)),
 	},
+	{
+		.callback = sof_rt5682_quirk_cb,
+		.matches = {
+			DMI_MATCH(DMI_PRODUCT_FAMILY, "Google_Rex"),
```

Comment:
Is there a reason why there is no DMI_OEM_STRING?

Reply:
Good question, for Rex, seems there is no other variant available, so DMI_OEM_STRING for AUDIO-XXX (FW_CONFIG) was not defined in the current coreboot code:

https://chromium.googlesource.com/a/chromiumos/third_party/coreboot/+/refs/heads/chromeos-2016.05/src/mainboard/google/rex/variants/rex0/overridetree.cb#218

Reply:
Can you take this up with CoreBoot folks @yongzhi1 @cujomalainey ? Thanks!

Reply:
Hi, @plbossart , just checked with our hw team, the only other I2S DB planned for Rex design is NAU88L25+ALC1019,
given that, think we are fine without the help of OEM string for this patch? Thanks!

Reply:
ok, I'll bet a beer this will change :-)